charts/ocis/values.yaml

```diff
@@ -907,191 +1890,412 @@ services:
-    #         matchExpressions:
-    #           - key: app
-    #             operator: In
-    #             values:
-    #               - appprovider
-    #       topologyKey: "kubernetes.io/hostname"
-    #
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # Do note that the value will be different for each service.
     # -- Per-service custom labels
     extraLabels: {}
     # Per-service image configuration. Overrides the default setting from `image` if set.
     image:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image repository
-      repository: ""
+      repository:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image tag.
-      tag: ""
+      tag:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image sha / digest (optional).
-      sha: ""
+      sha:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image pull policy
       pullPolicy:
 
   # -- APP REGISTRY service. Not used if `features.appsIntegration.enabled` equals `false`.
   # @default -- see detailed service configuration options below
   appregistry:
+    # @schema
+    # $ref: schema/templates.json#/resourcesTemplate
+    # @schema
     # -- Per-service resources configuration. Overrides the default setting from `resources` if set.
     resources: {}
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # -- Per-service nodeSelector configuration. Overrides the default setting from `nodeSelector` if set.
     nodeSelector: {}
+    # @schema
+    # type: [string, null]
+    # required: false
+    # @schema
     # -- Per-service priorityClassName configuration. Overrides the default setting from `priorityClassName` if set.
-    priorityClassName: ""
-    # -- Affinity settings for the appregistry service. See the documentation of this setting in approvider for examples.
+    priorityClassName:
+    # @schema
+    # $ref: schema/templates.json#/affinityTemplate
+    # @schema
+    # -- Per-service affinity.
     affinity: {}
+    # @schema
+    # $ref: schema/templates.json#/labelsTemplate
+    # @schema
     # -- Per-service custom labels
     extraLabels: {}
     # Per-service image configuration. Overrides the default setting from `image` if set.
     image:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image repository
-      repository: ""
+      repository:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image tag.
-      tag: ""
+      tag:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Image sha / digest (optional).
-      sha: ""
+      sha:
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
```

Comment:
We should add an enum here with the three values for `pullPolicy`

Reply:
same as https://github.com/owncloud/ocis-charts/pull/811/files#r1921904014